sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 * Sending a message twice will no longer result in a MessageAlreadySettled exception.
 * `ServiceBusClient.close()` now closes spawned senders and receivers.
 * Attempting to initialize a sender or receiver with a different connection string entity and specified entity (e.g. `queue_name`) will result in an AuthenticationError
+* Remove `is_anonymous_accessible` from management entities.
+* Remove `support_ordering` from `create_queue` and `QueueProperties`
+* Remove `enable_subscription_partitioning` from `create_topic` and `TopicProperties`
```

Comment:
a general question: we remove them because the service doesn't support them or we don't want to expose them?

Reply:
I must shamefully admit my context was limited to an ask re: cross-language-consistency.  We can follow up on this.

Reply:
Let's just apply the changes first